static/newsletter.json

```diff
@@ -0,0 +1,306 @@
+    "new_threads_this_week": [
+        {
+            "id": "022111",
+            "title": "The Pinning & Replacement Problem Set",
+            "link": "https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-November/022111.html",
+            "authors": [
+                "John Carvalho"
+            ],
+            "published_at": "2023-11-02T08:58:36+00:00",
+            "summary": "- John Carvalho critiques Bitcoin's fragility due to misbehaving miners and full blocks.\n- He advises Core to reconsider transaction policies for improved network harmony.\n- Carvalho offers solutions including reverting policies or allowing market-driven chaos.",
+            "n_threads": 3,
+            "dev_name": "bitcoin-dev",
+            "contributors": [
+                "Antoine Riard",
+                "Peter Todd"
+            ],
+            "file_path": "static/bitcoin-dev/Nov_2023/022111_The-Pinning-Replacement-Problem-Set.xml",
+            "combined_summ_file_path": "./static/bitcoin-dev/Nov_2023/combined_The-Pinning-Replacement-Problem-Set.xml"
```

Comment:
can we just make these 
`https://tldr.bitcoinsearch.xyz/summary/bitcoin-dev/Nov_2023/022111_The-Pinning-Replacement-Problem-Set`
and 
`https://tldr.bitcoinsearch.xyz/summary/bitcoin-dev/Nov_2023/combined_The-Pinning-Replacement-Problem-Set`
respectively.